lib/active_model/serializer.rb

```diff
@@ -143,29 +166,10 @@ def attributes(options = {})
       end
     end
 
-    def self.serializers_cache
-      @serializers_cache ||= ThreadSafe::Cache.new
-    end
-
-    def self.digest_caller_file(caller_line)
-      serializer_file_path = caller_line[CALLER_FILE]
-      serializer_file_contents = IO.read(serializer_file_path)
-      Digest::MD5.hexdigest(serializer_file_contents)
-    end
-
-    attr_reader :options
+    private # rubocop:disable Lint/UselessAccessModifier
```

Comment:
And yet

```ruby
 ActiveModelSerializers.silence_warnings do
      attr_accessor :instance_options
end
```

which is, I think, why rubocop thinks it's an unnecessary private